### ISSUE TITLE
ERROR: Error getting user details for wyll.bild. Please verify that the user exists.

### ISSUE THREAD

**Issue by Al3x1S4** [arc298/instagram-scraper#797]
Hi,

I run instagram-scraper as usual but I have only one answer wich is : 

ERROR: Error getting user details for wyll.bild. Please verify that the user exists.

I have tried with several users but the answers is always the same.

I have upgrade th program, uninstall it, install it again...

**Comment by AlexNik** [comment]
Do you scrape with authentiticacion?

**Comment by Ralfye** [comment]
No, but when I scrape with auth (-u xxx -p xxx), there is another error because there a dot (.) In the password so the code break. It doesn't work with ''

**Comment by AlexNik** [comment]
I see. 
I'll try to fix the bug with dot in the password.

**Comment by Ralfye** [comment]
Thks a lot :)
